kafka/client.go

```diff
@@ -25,6 +26,7 @@ type Client struct {
 	kafkaConfig   *sarama.Config
 	config        *Config
 	supportedAPIs map[int]int
+	topicsMutex   sync.Mutex
```

Comment:
Encountered a "concurrent map writes" panic during testing
map is not thread safe
Pre-existing issue